framework/src/actions/MeshOnlyAction.C

```diff
@@ -36,7 +36,7 @@ MeshOnlyAction::act()
   auto & mesh_ptr = _app.actionWarehouse().mesh();
 
   // Print information about the mesh
-  _console << mesh_ptr->getMesh().get_info(/* verbosity = */ 2);
+  _console << mesh_ptr->getMesh().get_info(/* verbosity = */ 2) << std::endl;
```

Comment:
I think this has an extra line already - flush instead?

Reply:
It is needed.  here is what it looks like now:

![image](https://user-images.githubusercontent.com/870705/123451100-a2539a80-d59a-11eb-96d4-5240609ce943.png)